vertx-pojo-mapper-common/src/main/java/de/braintags/io/vertx/pojomapper/dataaccess/query/IFieldCondition.java

```diff
@@ -0,0 +1,29 @@
+package de.braintags.io.vertx.pojomapper.dataaccess.query;
+
+/**
+ * A single condition statement of a query, to filter the result based on the value of a field<br>
+ * <br>
+ * Copyright: Copyright (c) 20.12.2016 <br>
+ * Company: Braintags GmbH <br>
+ * 
+ * @author sschmitt
+ */
+
+public interface IFieldCondition extends ISearchCondition {
+
+  /**
+   * @return the field of the condition
```

Comment:
See `UnknownQueryOperatorException` the constructor is not documented and contains a @`param` with the wrong parameter name